### PR TITLE
Atmospherics VD

### PR DESCRIPTION
**Pull request by pgranger23**
Adding the necessary config files to generate atmospheric neutrinos for the VD geometry
Depends on https://github.com/DUNE/dunesim/pull/73 to be merged first.

### CODE REVIEW

File: fcl/dunefdvd/gen/atm/prodgenie_atmnu_max_weighted_dunevd10kt_1x8x6_3view_30deg.fcl [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+outputs.out1.fileName: "prodgenie_atmnu_max_weighted_dunevd10kt_1x8x6_3view_30deg_gen.root"
+source.firstRun: 20000063
+
+physics.producers.generator: @local::dune_fdvd_genie_atmo_max_weighted_honda
+physics.producers.generator.EventsPerSpill: 1
+physics.producers.generator.POTPerSpill: 0
+# reduce the flux window to increase generation efficiency;
+# see `genie_dune.fcl` (or, better, GENIE manual) for explanations
+physics.producers.generator.Rl: 20 # meters
+physics.producers.generator.Rt: 20 # meters
```
Comment (absolution1) [inline fcl/dunefdvd/gen/atm/prodgenie_atmnu_max_weighted_dunevd10kt_1x8x6_3view_30deg.fcl:32]: Could these be defined in genie_dune?

**Review by absolution1 — COMMENTED** [review]
Hi @pgranger23 
Thanks for adding this.  I had one question about the location of the parameter overrides.

File: fcl/dunefdvd/gen/atm/prodgenie_atmnu_max_weighted_dunevd10kt_1x8x6_3view_30deg.fcl (hunk 1 above)
Comment (pgranger23) [reply fcl/dunefdvd/gen/atm/prodgenie_atmnu_max_weighted_dunevd10kt_1x8x6_3view_30deg.fcl]: Should be done by now

**Review by absolution1 — APPROVED** [review]
Perfect, thank you!

**Pull request merged**